workflow/header/nsc_header.txt

```diff
@@ -0,0 +1,12 @@
+# parent_id: 'spp'
```

Comment:
Another possibility to try is to unquote all `parent_id` and `id` fields, e.g. for this one this would be:
```suggestion
# parent_id: spp
```